_plugins/redirector.rb

```diff
@@ -30,20 +30,11 @@ def initialize(site, group, redirectConfig, mapping)
         @name     = basename + ext # basically @basename + @ext.
         @layout  = 'redirect.html'
 
-        @content = "#{redirectConfig['baseUrl']}#{mapping['path']}"
-
-        # Initialize data hash with a key pointing to all posts under current category.
-        # This allows accessing the list in a template via `page.linked_docs`.
         @data = {
-          'version' => mapping['fromVersion'],
-          'target' => redirectConfig['baseUrl'] + mapping['path'],
+          'target' => redirectConfig['baseUrl'] + mapping['fromVersion'] + mapping['path'],
```

Comment:
wondering what we want here, should v0.13.0 of the code reference a link with the version embedded in it? So that we can link to the appropriate documentation version? (imagining that maybe the TLS configuration has evolved since 0.13.0 and the latest docs wouldn't work for their version)

If we did that we would have to have some way to generate a link for each new version.